src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  font-family: "aventir", sans-serif;
+  font-weight: 500;
+  color: #3d4e61;
+  margin-top: 0;
+  box-sizing: border-box;
+}
```

Comment:
Don't set any styles except box sizing inside `*` selector
put font styles to `html` selector and inherit them for elements where it needed